e2e/pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
-
```

Comment:
not sure why this changed, what's your guys' pnpm version?

Reply:
8.6.12, `pnpm install --force` doesn't change lockfile for me

Reply:
8.6.8

Reply:
upgraded to `8.6.12` and the diff is gone 👍